### ISSUE TITLE
Is the fieldset hint guidance correct?

### ISSUE THREAD

**Issue by joelanman** [alphagov/govuk-design-system#2143]
## Related documentation
https://design-system.service.gov.uk/components/fieldset/

## Suggestion
It reads
> Include any general help text which is important for filling in the form and cannot be written as [hint text](https://design-system.service.gov.uk/components/text-input/#hint-text) in the legend, but try to keep it as concise as possible.

This would make help text look like a heading, which I'm not sure is correct? If it is, it would be useful to see an example.

**Comment by davidc-gds** [comment]
My quick thought on this: Instead of adding the hint-text type of text into the legend, should we be working to add it as a <p> after the legend, and before the first field? That might mean it gets a bit lost though for screen reader users who are tabbing between fields.

**Comment by EoinShaughnessy** [comment]
Great catch, thanks @joelanman. My hunch is there should have been a comma after 'hint text'. The sentence is overlong anyway so I'll try rewriting it.

**Comment by joelanman** [comment]
on support, the context was asking for an address, with the first field in the fieldset being the address, then postcode and country. In that context a hint attached to the first field I think could work. I'm not sure this is generalisable to all fieldsets. It has the advantage of getting read out by screen readers
